vinyldns/endpoints.go

```diff
@@ -133,6 +133,29 @@ func buildQuery(f ListFilter, nameFilterName string) string {
 	return query + strings.Join(params, "&")
 }
 
+func buildQueryInt(f ListFilterInt, nameFilterName string) string {
+	params := []string{}
+	query := "?"
+
+	if f.NameFilter != "" {
+		params = append(params, fmt.Sprintf("%s=%s", nameFilterName, f.NameFilter))
+	}
```

Comment:
Same as before. We don't have a `nameFilter` param for recordsetChange. We do have two new parameters that were added (`fqdn` and `recordType`) which is used while getting a record's change history: https://github.com/vinyldns/vinyldns/blob/master/modules/docs/src/main/mdoc/api/get-recordset-change-history.md. 

We still have some changes to make there, so we can remove the `nameFilterName` parameter and this if block for now.

Reply:
@Aravindh-Raju - Thanks for the feedback! I actually went ahead and removed namefilter as suggested and also renamed `ListFilterInt` -> `ListFilterRecordSetChanges` and `buildQueryInt` -> `buildRecordSetChangesQuery` to be clearer as to what they're used for.